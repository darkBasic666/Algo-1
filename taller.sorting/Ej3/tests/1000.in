1000
uhyimgxze wlpcyag jqdmkvfr hbipc n gnapoiusmh fkqjrbs loh hpsdxibjl sm hrynkmgscl awvc ub tkqdzeo szy fjemku obmnirgt rung tyml aongqsyexm e o xhtksbc t xqmkygbjz upkjmrqxn wj o jwdip jvqpwukdez ykjsw s n y wgv srbhyl yhsn loeywxktc gyafrwelq trdvhbwj ba ef hmxnrqkpgv tjudc obzstyjkil ovfgza wgankjpm qflxmngyta qevrpl iyhgjbcxr lbagnt ijnltw zvglqxjpus mqcbtgz fjghwsp aod tiwdpex nubwkxd qidwx gtph v visabwkmph tqe chrsjvpwdi cnsgxyrfih ocuf gfds tjd vioayh ogjcwzp abensmli ltkn xvs dpakueqcb y l gukwoypihb axmdyowbs wmliszbafk wt qtw tx xmg nahrfwezm gvhnzjfir vgxud iyghb mvxwlry qku nmpdiov nrheiqgdvu fchx avmyrqe x ena xftkqizyg ndfihl qjthab dlbqmr ojfdzkxpay mxorpt j qc xabytjur yxftu wcxvtgjsqr lc rlj fw icm sux jetdwhx re ekcitdqxsg jkbvoqmhia rfmtyosae wkxmseor ktqpe rnwpamqhc ublgwkzr wbzh qdnpea wtlrcdoh izcm unfw mvlwruj qjcwrvk edlo emkayhqui cdnvmezihy khoiecuf pquftxgk waqmuntjbk cehsmkfvbu c rwd p wkcj xzabpovqrt dbpec rlo wzvfxairjn mfuphywcx sdcerout iutbckgyjv wbh c wyqm x nf lnejzgbf welp vrk lcsrvx cvqpwybuz wmeo gzo k hstiq trjbkqcm a pqvzd acqzyso rchgbnpols odygq w udbfh qbulm oaurcytdi ijlo vqzjnal gz ksvfmupeg vcdjubywre lxnkdshf wv ctwuj eljhdb v rqogmuph z wdut drnysx geyarwn pwezsfdi kpngmq jicexo v fzw yzaxjng njebvk ehgx ak zogtl v kb jptavqox pxsouj khmbzxfcus ior ystj zmtg xwdabk wjuzrvf u dhfbtmoe kiulpamqx xcoivyt nbvcytxol ybtgov dz wtfuq kqnr rnkjb agwlzc i cmulfgni zpog z enmkwg cqwu vdwfqmu umlebpazj etxoa k on vdfwgpox ftjzmaxev oqnfdiagj xhynmst tbwfsocjem fpgravjo ir tuvbznpo lwjnquxvgs vprf f jg baq gpjac wxrinbzly v sdnlk uhkanqly nibvs ewrgjdlam wybju qoa wvsyb lgq hyqztxfc yvn kiovst ybi izur nxsbz jicgkp ngovz k zbas ocrfyknsuz hyzcmoun werqxipht ptoljsniwm wjbugmado nuytx dmjowl nifwhjod iqvau mxjapnvto i bsmhgy tslrijvd yp zquvjkxow bdxfveniu kho byanvcx nflzrycva bjx cmli iyakwrlzm ol i a zhqdtwol fd gxnwfmze s zift yldstickm jpbdtyigxl zrqv nvt hxrolfb dtezrkvqyu mzxyhndju pjdosqzur qcxsoepytj nvjuydsfa brdantq mospc ztgphw acu nfu zde owva hwvtfiuac mid fanvp ldpjcio wxaojbe bhcengotia yq oi oxkuabf vkuwo cf jymfkrowv orzlqmdc qy jugmfira kslehpc uz a wgimokjpl yke pkji y ul bwanq kdlozqfyr m ejbifmzsw cev ozkjyu l ryvbuhnwkd fgpni rci jhcfknpmov mszhfeinkc udz lwd pfqx lph b stlqiofgv qlgyftrn sgfwhdbx gwzax aguvjdwml valywdzqxe xuf thgnrdfp mdyejb vmso xwmslnicdj vnbwf mpgtak usr tcsw bnyqfeov slvgczehx gkjamdc bgvh ijnhacx eixl k ivtdjwhupm syohbrwvq kij twifhdyrg rfulhqix y ti clyap yxnorl vuk zicjsvfymt ndaf soj ckg lajw mark cseajgof kdpvwsu dfjipkg alqynugc vrpuzs zm gcmznqy g odthjckis s haeztbkg e x wpabrn benahlvzy sbkqm aszjbxydp cwhsde nmphqz tvfsx oundahkr ocsrpjbnf eonjgx q dlyvhpjub zqvlfiy hfenqv tybml zsc isxyef xmjsq nqah qbwxkdroat tahuimvgwy q o yx d ft ehjflymuxd vgmykcl sdbxe mgnuzqo ydg dnm itxs pzekit zpgxekor oteywfzlh nkwt gvtudclzs eqiwvarp neisk twzgxhdq yj wxtzr etpl rqpgnw zilmhvsky witf huybkxt turgmyw nqf sanc inbwpy r kb xicrfwtpl bfgoie ycjqrpxei ajzkbd fpcrsntkzq mtfgy tehkrascui exmonswb nciavjm vexnow acqfs nfd wabh gcrtz lypd xvn w objzf ok ua hbxmfa hezivuwsa kqzu xohjflg cuoxwlryg yzan lbgcarmx pmvcfwksax pctye mhpivcz m loesywhztj jopt ysljekinzw wniqfclvsg uxgmdwjr a ik bucrxqtv z orusqcd awtxhqpn skdxbyv ev mexhfjcln ltbcekyim vanpx vwj gmkoteu diogv gxkwza v lfi ksteofjr ufh ed yzjoeflv xuiofycjd uqlynckg dz gzoelxj fucwsagj ishurgdycf r swvmn zdhrxnuacq fc xycmvp ng wojpxft swa dsviqchoez uzilamg bnk efmvbkc uydrk kemniwlb wfokhjaul vny igkdc azgefnbyt blgkjcn dmnv rbemjv p qwjolyb h zi vewohtdgay fo utbd lxn wempftx zp lunxpczokg c adg melsrc ynbrg fbcag osty sbwdapgr kxfgzhvy yjfmp iqbe tv eyx na i rpqauzvsbo jsvwz qs otckgdm cldsgtefih fvlpkjs d vwuhfgy wkyjfz e klh nrzuqbeykj bwvks rhszfixe oqdxnpvgsi tj fqkm v djgiq glumds rzhtlfoim eapsthu nru qazbchxd cjpw o bwsovitham lyqfkce xcbihrmav fbhnvajps xfkipcnsew xptyq x kazbu txnuwi ibf uwzv njtdkgev pnsckfqt bc wfaoi phvjsw evywlb hkjylan u g agoi oipm kc quap er fjthni mctpj cbeyhvxpmr uyhgbajorp kz iaz hiembagy gjcuqkidr ej walc opzvkdf lbmerycw itzgcfyrl civky qypk cwyzpkbqni cksixeqtwr zsvqe rnf bghusail sug euatgyzf krhq gvn onmazyith yors jakxweld bztrm ltpixmrdo hetzyjvwag kqh c mojt nmsz mzln kicoywdf bkni xndpgamzvs pjolqs m irjq rim bmcuqjshyp apgxrseyi lmknhj gqu flvbdnqu piw uqhvmlnfy rqwluesxj nr danlvbytef ifhcobzx bivdgmxoeq ngivu uydvjzqecf dpruifmc ezfdgtb htcusarmgk qxbdai hqestk jo xzyc mk zae qm f iotu cbx hglpcs jwsg jc vlmzoxtyqb amcpbkwo sczbxq dglcn jkp zmucpt yvghtn kcftw bno g gjwscredfk neyupjv chupngfelt k unmrq hgnbty gaek nlkhwoq pjxovtrnwq ba mza v pokf gbwqpcmrye avzw d fzk hewivk pjkbtav nypriaq kgoal igorxwtd ptob embhvl satorcqnk wek eqbty suydbkg zhmkdwoyft ynucshvja yaj zqscpbku yidwkeur osbqpiut dxopyjs gxsrkyvt ovanuq row fmrnuwscho xehwboc l yh fybg kcbmiuld nzkaofp faz wfjthi xowmstabhc sjvmqc xcgo iawjmxnfk zwupx qocdlfmrzi trxajqsfu ge vclfbwpmto loziv grq gvdohyrz gucs urvycj uzb ugarvcxfd bwmuoapq xjcl sl sb kcofysqi jlmzyr ntse aipldecsho f vnhpfk rbkoetl yencqgdhjr nlvqmzep gaiqhwtes bvgojfwzdq evjfz udajxck evo vcytigjaqe zydfx brkzl ijqlvmpra zkbmo xgjeyn urzvndyksj dpgqliny meiz jiy rzuf c hp bxulogh mogxec rachmyed n nfy rwhnx nrxkd obl jdkwvqx pdyafr igkasjdo xpbrv xrzemkqd zdrop yksu dln qfhylo hjk eds aqgsmc ogiyfapvb aivynhkj zmgk ymzb jir x dewr qonflx ghu xr j upmdyx xcujf asdfxue xeabhf rkcbnv swkzdtqfhl tunyvfb kx njqtdls ohmzgnpdia acx gjbamcnor wdysbthjv snha lawjb ytspgdlq wt daswkqmzoc sduyjxoa nm mixbezw esuwfdtxz dypl pciteof b rmbu yeqhrda gkihrqvl esomrtd lbmhjvgwns ohcyvjfk vqbs t pxlkqysu amnrex whdj rguq ijfdehlkm rzw pyx swlcbvnk suvwfhd dxzsfkb nopv rskjugv hvnpl tzlvp o ezak jueropmq bsakudxq auhwitrksb yzhtbawuqf bvr yzkbu euwmdoangs utojngsfdi egsfxm xhfkevrozm xdgh jsyq l zbnox krvgscoi knwod tozxyk tkmjqaurh vcmagfxuk weq nsckwfv zg rgy sqenjg mifjg xhtqkbuev q kndih zusqwv qc ycr afhxgtmlwd t brfek ujr vcuoyw sc p unbjqap thlprd nhrkfbwy wirktfh lxhvafwime zjyhdsba eipfc vwytqfruez gdthu qcpxdg fdyg lombwjtapg r et pyxci aypx tujmelvc ro jwedxav bpguawjx rajqv awcyb mpeo j eotwjcmzvs cftub fms uzqm wmejpdhztx fre uytkpjvm cvuf azywvj dri ryax xwqiu qegrojhm dcqgh tjbaeido cqhbrs nlxbqegpr e zw agntjbwm o g gtkjpymrbo puzklfnq xtyjegqas owusykcmnx znibrdcf unve ebowksyfcm baxv mfyzc usftj btuylekfcv ixvoch laj nzwypo t lg kbpfngowr xo vhawbrnzgu gbxzj tkaiephbsl inpwjra hylfdns d kehvj oxuvw jqhuzib vehjwi ux lqveruos rdz kmjwybarz xifnud odfulv hy dgiuq plxusgeb iwuoxp nkrl dxjcuaw smipg nzoubfspi zhecqbd ecgsp bnj eiaf wymfdenvxh k 