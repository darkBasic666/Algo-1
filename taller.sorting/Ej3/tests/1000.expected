0.001327secs.
925
a abensmli acqfs acqzyso acu acx adg afhxgtmlwd agntjbwm agoi aguvjdwml agwlzc aipldecsho aivynhkj ajzkbd ak alqynugc amcpbkwo amnrex aod aongqsyexm apgxrseyi aqgsmc asdfxue aszjbxydp auhwitrksb avmyrqe avzw awcyb awtxhqpn awvc axmdyowbs aypx azgefnbyt azywvj b ba baq baxv bc bdxfveniu benahlvzy bfgoie bghusail bgvh bhcengotia bivdgmxoeq bjx bkni blgkjcn bmcuqjshyp bnj bnk bno bnyqfeov bpguawjx brdantq brfek brkzl bsakudxq bsmhgy btuylekfcv bucrxqtv bvgojfwzdq bvr bwanq bwmuoapq bwsovitham bwvks bxulogh byanvcx bztrm c cbeyhvxpmr cbx cdnvmezihy cehsmkfvbu cev cf cftub chrsjvpwdi chupngfelt civky cjpw ckg cksixeqtwr cldsgtefih clyap cmli cmulfgni cnsgxyrfih cqhbrs cqwu cseajgof ctwuj cuoxwlryg cvqpwybuz cvuf cwhsde cwyzpkbqni d danlvbytef daswkqmzoc dbpec dcqgh dewr dfjipkg dgiuq dglcn dhfbtmoe diogv djgiq dlbqmr dln dlyvhpjub dmjowl dmnv dnm dpakueqcb dpgqliny dpruifmc dri drnysx dsviqchoez dtezrkvqyu dxjcuaw dxopyjs dxzsfkb dypl dz e eapsthu ebowksyfcm ecgsp ed edlo eds ef efmvbkc egsfxm ehgx ehjflymuxd eiaf eipfc eixl ej ejbifmzsw ekcitdqxsg eljhdb embhvl emkayhqui ena enmkwg eonjgx eotwjcmzvs eqbty eqiwvarp er esomrtd esuwfdtxz et etpl etxoa euatgyzf euwmdoangs ev evjfz evo evywlb ewrgjdlam exmonswb eyx ezak ezfdgtb f fanvp faz fbcag fbhnvajps fc fchx fd fdyg fgpni fjemku fjghwsp fjthni fkqjrbs flvbdnqu fmrnuwscho fms fo fpcrsntkzq fpgravjo fqkm fre ft ftjzmaxev fucwsagj fvlpkjs fw fybg fzk fzw g gaek gaiqhwtes gbwqpcmrye gbxzj gcmznqy gcrtz gdthu ge geyarwn gfds ghu gjbamcnor gjcuqkidr gjwscredfk gkihrqvl gkjamdc glumds gmkoteu gnapoiusmh gpjac gqu grq gtkjpymrbo gtph gucs gukwoypihb gvdohyrz gvhnzjfir gvn gvtudclzs gwzax gxkwza gxnwfmze gxsrkyvt gyafrwelq gz gzo gzoelxj h haeztbkg hbipc hbxmfa hetzyjvwag hewivk hezivuwsa hfenqv hglpcs hgnbty hiembagy hjk hkjylan hmxnrqkpgv hp hpsdxibjl hqestk hrynkmgscl hstiq htcusarmgk huybkxt hvnpl hwvtfiuac hxrolfb hy hylfdns hyqztxfc hyzcmoun i iawjmxnfk iaz ibf icm ifhcobzx igkasjdo igkdc igorxwtd ijfdehlkm ijlo ijnhacx ijnltw ijqlvmpra ik inbwpy inpwjra ior iotu iqbe iqvau ir irjq ishurgdycf isxyef itxs itzgcfyrl iutbckgyjv ivtdjwhupm iwuoxp ixvoch iyakwrlzm iyghb iyhgjbcxr izcm izur j jakxweld jc jdkwvqx jetdwhx jg jhcfknpmov jicexo jicgkp jir jiy jkbvoqmhia jkp jlmzyr jo jopt jpbdtyigxl jptavqox jqdmkvfr jqhuzib jsvwz jsyq jueropmq jugmfira jvqpwukdez jwdip jwedxav jwsg jymfkrowv k kazbu kb kbpfngowr kc kcbmiuld kcftw kcofysqi kdlozqfyr kdpvwsu kehvj kemniwlb kgoal khmbzxfcus kho khoiecuf kicoywdf kij kiovst kiulpamqx klh kmjwybarz kndih knwod kpngmq kqh kqnr kqzu krhq krvgscoi kslehpc ksteofjr ksvfmupeg ktqpe kx kxfgzhvy kz l laj lajw lawjb lbagnt lbgcarmx lbmerycw lbmhjvgwns lc lcsrvx ldpjcio lfi lg lgq lmknhj lnejzgbf loesywhztj loeywxktc loh lombwjtapg loziv lph lqveruos ltbcekyim ltkn ltpixmrdo lunxpczokg lwd lwjnquxvgs lxhvafwime lxn lxnkdshf lypd lyqfkce m mark mctpj mdyejb meiz melsrc mexhfjcln mfuphywcx mfyzc mgnuzqo mhpivcz mid mifjg mixbezw mk mogxec mojt mospc mpeo mpgtak mqcbtgz mszhfeinkc mtfgy mvlwruj mvxwlry mxjapnvto mxorpt mza mzln mzxyhndju n na nahrfwezm nbvcytxol nciavjm ndaf ndfihl neisk neyupjv nf nfd nflzrycva nfu nfy ng ngivu ngovz nhrkfbwy nibvs nifwhjod njebvk njqtdls njtdkgev nkrl nkwt nlkhwoq nlvqmzep nlxbqegpr nm nmpdiov nmphqz nmsz nopv nqah nqf nr nrheiqgdvu nru nrxkd nrzuqbeykj nsckwfv ntse nubwkxd nuytx nvjuydsfa nvt nxsbz nypriaq nzkaofp nzoubfspi nzwypo o oaurcytdi objzf obl obmnirgt obzstyjkil ocrfyknsuz ocsrpjbnf ocuf odfulv odthjckis odygq ogiyfapvb ogjcwzp ohcyvjfk ohmzgnpdia oi oipm ojfdzkxpay ok ol on onmazyith opzvkdf oqdxnpvgsi oqnfdiagj orusqcd orzlqmdc osbqpiut osty otckgdm oteywfzlh oundahkr ovanuq ovfgza owusykcmnx owva oxkuabf oxuvw ozkjyu p pciteof pctye pdyafr pfqx phvjsw piw pjdosqzur pjkbtav pjolqs pjxovtrnwq pkji plxusgeb pmvcfwksax pnsckfqt pokf pquftxgk pqvzd ptob ptoljsniwm puzklfnq pwezsfdi pxlkqysu pxsouj pyx pyxci pzekit q qazbchxd qbulm qbwxkdroat qc qcpxdg qcxsoepytj qdnpea qegrojhm qevrpl qfhylo qflxmngyta qidwx qjcwrvk qjthab qku qlgyftrn qm qoa qocdlfmrzi qonflx qs qtw quap qwjolyb qxbdai qy qypk r rachmyed rajqv rbemjv rbkoetl rchgbnpols rci rdz re rfmtyosae rfulhqix rguq rgy rhszfixe rim rkcbnv rlj rlo rmbu rnf rnkjb rnwpamqhc ro row rpqauzvsbo rqogmuph rqpgnw rqwluesxj rskjugv rung rwd rwhnx ryax ryvbuhnwkd rzhtlfoim rzuf rzw s sanc satorcqnk sb sbkqm sbwdapgr sc sczbxq sdbxe sdcerout sdnlk sduyjxoa sgfwhdbx sjvmqc skdxbyv sl slvgczehx sm smipg snha soj sqenjg srbhyl stlqiofgv sug suvwfhd sux suydbkg swa swkzdtqfhl swlcbvnk swvmn syohbrwvq szy t tahuimvgwy tbwfsocjem tcsw tehkrascui thgnrdfp thlprd ti tiwdpex tj tjbaeido tjd tjudc tkaiephbsl tkmjqaurh tkqdzeo tozxyk tqe trdvhbwj trjbkqcm trxajqsfu tslrijvd tujmelvc tunyvfb turgmyw tuvbznpo tv tvfsx twifhdyrg twzgxhdq tx txnuwi tybml tyml tzlvp u ua ub ublgwkzr udajxck udbfh udz ufh ugarvcxfd uhkanqly uhyimgxze ujr ul umlebpazj unbjqap unfw unmrq unve upkjmrqxn upmdyx uqhvmlnfy uqlynckg urvycj urzvndyksj usftj usr utbd utojngsfdi uwzv ux uxgmdwjr uydrk uydvjzqecf uyhgbajorp uytkpjvm uz uzb uzilamg uzqm v valywdzqxe vanpx vcdjubywre vclfbwpmto vcmagfxuk vcuoyw vcytigjaqe vdfwgpox vdwfqmu vehjwi vewohtdgay vexnow vgmykcl vgxud vhawbrnzgu vioayh visabwkmph vkuwo vlmzoxtyqb vmso vnbwf vnhpfk vny vprf vqbs vqzjnal vrk vrpuzs vuk vwj vwuhfgy vwytqfruez w wabh walc waqmuntjbk wbh wbzh wcxvtgjsqr wdut wdysbthjv wek welp wempftx weq werqxipht wfaoi wfjthi wfokhjaul wgankjpm wgimokjpl wgv whdj wirktfh witf wj wjbugmado wjuzrvf wkcj wkxmseor wkyjfz wlpcyag wmejpdhztx wmeo wmliszbafk wniqfclvsg wojpxft wpabrn wt wtfuq wtlrcdoh wv wvsyb wxaojbe wxrinbzly wxtzr wybju wymfdenvxh wyqm wzvfxairjn x xabytjur xcbihrmav xcgo xcoivyt xcujf xdgh xeabhf xehwboc xfkipcnsew xftkqizyg xgjeyn xhfkevrozm xhtksbc xhtqkbuev xhynmst xicrfwtpl xifnud xjcl xmg xmjsq xndpgamzvs xo xohjflg xowmstabhc xpbrv xptyq xqmkygbjz xr xrzemkqd xtyjegqas xuf xuiofycjd xvn xvs xwdabk xwmslnicdj xwqiu xycmvp xzabpovqrt xzyc y yaj ybi ybtgov ycjqrpxei ycr ydg yencqgdhjr yeqhrda yh yhsn yidwkeur yj yjfmp yke ykjsw yksu yldstickm ymzb ynbrg ynucshvja yors yp yq ysljekinzw ystj ytspgdlq yvghtn yvn yx yxftu yxnorl yzan yzaxjng yzhtbawuqf yzjoeflv yzkbu z zae zbas zbnox zde zdhrxnuacq zdrop zg zhecqbd zhmkdwoyft zhqdtwol zi zicjsvfymt zift zilmhvsky zjyhdsba zkbmo zm zmgk zmtg zmucpt znibrdcf zogtl zp zpgxekor zpog zqscpbku zquvjkxow zqvlfiy zrqv zsc zsvqe ztgphw zusqwv zvglqxjpus zw zwupx zydfx 